client/landing/stepper/declarative-flow/internals/steps-repository/trial-acknowledge/index.tsx

```diff
@@ -9,14 +9,14 @@ const TrialAcknowledge: Step = function TrialAcknowledge( { navigation, flow, st
 	const { goBack } = navigation;
 
 	const getStepContent = () => {
-		if ( isAnyMigrationFlow( flow ) ) {
+		if ( flow === NEW_HOSTED_SITE_FLOW ) {
```

Comment:
Let's use the helper function:

```suggestion
		if ( isNewHostedSiteCreationFlow( flow ) ) {
```

Reply:
done 👍